accesspoc/templates/base.html

```diff
@@ -35,24 +35,16 @@
               <span class="sr-only">{% trans "Select language" %}</span>
             </a>
             <div class="dropdown-menu dropdown-menu-right" aria-labelledby="langMenu">
-              {% get_current_language as LANGUAGE_CODE %}
-              {% get_available_languages as LANGUAGES %}
-              {% get_language_info_list for LANGUAGES as languages %}
-              <form action="{% url 'set_language' %}" method="post" class="px-3 py-2">
+              <form name="langForm" action="{% url 'set_language' %}" method="post">
                 {% csrf_token %}
                 <input name="next" type="hidden" value="{{ redirect_to }}" />
-                {% for language in languages %}
-                  <div class="form-group form-check">
-                    <input class="form-check-input" type="radio" name="language" id="lang_{{ language.code }}" value="{{ language.code }}"{% if language.code == LANGUAGE_CODE %} checked="checked"{% endif %}>
-                    <label class="form-check-label" for="lang_{{ language.code }}">
-                      {{ language.name_local|capfirst }} ({{ language.code }})
-                    </label>
-                  </div>
-                {% endfor %}
-                <div class="text-right">
-                  <button type="submit" class="btn btn-sm btn-primary">{% trans "Submit" %}</button>
-                </div>
+                <input name="language" type="hidden" />
               </form>
+              {% get_available_languages as LANGUAGES %}
+              {% get_language_info_list for LANGUAGES as languages %}
+              {% for language in languages %}
+                <a class="dropdown-item" href="javascript:;" onclick="document.langForm.language.value='{{ language.code }}'; document.langForm.submit();">{{ language.name_local|capfirst }} ({{ language.code }})</a>
```

Comment:
You came this far without a single .js file but perhaps you should have one now to avoid mixing code and markup. It used to be a bad practice IIRC.